langwrappers/src/main/kotlin/org/jetbrains/research/testspark/langwrappers/PsiComponents.kt

```diff
@@ -90,6 +90,14 @@ interface PsiClassWrapper {
 interface PsiHelper {
     val language: SupportedLanguage
 
+    /**
+     * Checks if a class is available for generation at the given caret offset.
```

Comment:
> a class is available

I think you meant whether a valid code construct under the current cursor exists, for which it's possible to generate tests. 

You can write:
```java
/**
* Checks if a code construct is valid for unit test generation at the given caret offset
* */
```

Reply:
updated